### PR TITLE
Fix typos for overview on the homepage.md 📕✏️

### PR DESCRIPTION
**Pull request by debashiscodes**
Fix typos for **Overview** on the **homepage.md**.

- changed the open source to the open-source
- Changed the `requirements` tools to the `required` tools.

### CODE REVIEW

File: docs/content/homepage.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ slug: /
 hide_table_of_contents: true
 ---
 
-Welcome to Medusa, the open source Shopify alternative!
+Welcome to Medusa, the open-source Shopify alternative!
```
Comment (shahednasser) [inline docs/content/homepage.md:9]: We actually deliberately use "open source" so please change it back 🙂

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for noticing the typos! Just a small comment below

File: docs/content/homepage.md (hunk 1 above)
Comment (DebashisCodes) [reply docs/content/homepage.md]: @shahednasser Done ✅ . Please have a look.

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Thanks @DebashisCodes 🚀

**Pull request merged**